README.md

```diff
@@ -64,6 +80,17 @@ Bugs and feature requests should be raised on the [issue page](https://github.co
 
 > **_NOTE:_**  We'd love more contributors to this project!
 
+## How to Contribute
+
+Interested in contributing to autodE? We welcome contributions from the community, be it in the form of bug reports, feature requests, documentation improvements, or code contributions. Here's how you can get started:
+
+Check out our [Issues] (https://github.com/duartegroup/autodE/issues) for known bugs or feature requests.
+**Fork** the repository and make your changes.
+Submit a **Pull Request** with a detailed description of the changes.
+Participate in the discussion until your PR is merged.
+For more information, join our [Slack workspace] (https://autodeworkspace.slack.com) (email to be added) or consult the CONTRIBUTING.md guide in our repository.
```

Comment:
This is covered in https://duartegroup.github.io/autodE/dev/contributing.html – it probably needs updating, and  linked to more prominently from the here